### ISSUE TITLE
Not possible to log actual underlying Docker images

### ISSUE THREAD

**Issue by brandonros** [binci/binci#99]
We use Binci in our CI/CD. At like a 30% random rate, services fail to start sometimes.

https://github.com/binci/binci/blob/master/src/services.js#L86 calls https://github.com/binci/binci/blob/master/src/proc.js#L13 but it does it with `silent: true` and no way to override it.

Maybe some debugging options for errors?

**Comment by Fluidbyte** [comment]
This is an issue I've been wavering on since I started this project. Typically (at least for myself and teams I've used Binci with) a failed service start is a matter of configuration. 

Since under the hood Binci essentially just constructs a series of `docker run ...` commands then links them on the "primary" container, a service failure seems easiest to debug by simply running that service and/or tailing docker logs.

That being said, it definitely would be preferable to have some sort of output. I feel like it would be best to have this as a `verbosity` level control so `info` level stuff doesn't output unless explicitly requested, overriding the default `silent: true` for errors caught and spitting them out.
